articles/connections/database/custom-db/create-db-connection.md

```diff
@@ -0,0 +1,152 @@
+---
+description: Learn how to create a database connection.
+toc: true
+topics:
+    - connections
+    - custom-database
+contentType: how-to
+useCase: customize-connections
+---
+Create an Auth0 database connection
+
+The first thing you will do is create a database connection in Auth0:
+
+1. Log in to the Dashboard and navigate to [Connections > Database](${manage_url}/#/connections/database).
+
+2. Click **+ Create DB Connection**.
+
+3. Configure the connection's **settings** as requested.
+
+| **Parameter** | **Definition** |
+| - | - |
+| **Name** | The name of the connection. The name must start and end with an alphanumeric character, contain only alphanumeric characters and dashes, and not exceed 35 characters. |
+| **Requires Username** | Forces users to provide a username *and* email address during registration. |
+| **Username length** | Sets the minimum and maximum length for a username. |
+| **Disable Sign Ups** | Prevents sign-ups to your application. You will still be able to create users with your API credentials or via the Dashboard, however. |
+
+4. Click **Create** to proceed.
+
+![Database connections](/media/articles/connections/database/database-connections.png)
+
+Once Auth0 creates your connection, you'll have the following tabs (in addition to the **Settings** tab):
+
+* Password Policy
+* Custom Database
+* Applications
+* Try Connection
+
+4. Switch over to the **Custom Database** tab.
+
+5. Toggle the **Use my own database** switch to enable this feature.
+
+![Custom database tab](/media/articles/connections/database/custom-database.png)
+
+## Step 2: Create database action scripts
```

Comment:
Looks like there should be a header before that for `Step 1`, we can't start from 2